src/community/security/keycloak/src/main/java/org/geoserver/security/keycloak/GeoServerKeycloakFilter.java

```diff
@@ -291,11 +301,8 @@ protected AuthResults loadAuthn(HttpServletRequest request) {
 
     private static void logHttpRequest(Level level, HttpServletRequest request) {
         if (LOG.isLoggable(level)) {
-            LOG.log(level, "request.method   = " + request.getMethod());
```

Comment:
Why was the logging removed here?

Reply:
Ah, I had put a reasonable amount it to resolve and issue with the redirects, I might have cleaned up this one by mistake. I will reinstated